ubcpi/templates/ubcpi.html

```diff
@@ -0,0 +1,290 @@
+                            <p role="alert" class="warning" id="button-disabled-reason">
+                                <span class="sr" translate>Warning</span>
+                                <i aria-hidden="true" class="icon fa fa-warning"></i>&nbsp;<span translate>Note:</span>
+                                <span ng-if="answerForm.rationale.$error.minlength" translate>Your rationale must be a minimum of {{rationale_size.min}} characters.</span>
+                                <span ng-if="answerForm.rationale.$error.maxlength" translate>Your rationale must be a maximum of {{rationale_size.max}} characters.</span>
+                                <span ng-if="answerForm.rationale.$invalid && answerForm.q.$valid" translate>In order to move to the next step please briefly explain why you chose this answer.</span>
+                                <span ng-if="answerForm.rationale.$valid && answerForm.q.$invalid" translate>In order to move to the next step please choose an answer.</span>
+                                <span ng-if="answerForm.q.$invalid && answerForm.rationale.$invalid" translate>In order to move to the next step, please choose an answer and briefly explain your choice.</span>
+                            </p>
+                        </div>
+                    </div>
+                {% endif %}
+                <div class="ubcpi-staff-button-wrapper" ng-if="user_role == 'instructor' || user_role == 'staff'" data-ng-init="rc.getStats()">
+                    <a ng-model="collapse" ng-click="collapse=!collapse" ng-class="{true: 'ubcpi-staff-statistics-button--active', false: 'ubcpi-staff-statistics-button'}[collapse]" translate>View Question Statistics</a>
+                    <div class="ubcpi-staff-statistics" ng-show="collapse">
+
+                        <div class="original-answers results-container choicegroup">
+                            <pi-barchart options="options" stats="rc.stats.original" correct="rc.correct_answer" data-role="user_role"></pi-barchart>
+                            <p class="ubcpi-chart-label">Original Answer</p>
+                        </div><!-- .original-answers -->
+
+                        <div class="revised-answers results-container choicegroup">
+                            <pi-barchart options="options" stats="rc.stats.revised" correct="rc.correct_answer" data-role="user_role"></pi-barchart>
+                            <p class="ubcpi-chart-label">Revised Answer</p>
+                        </div><!-- .revised-answers -->
+                        </div>
+                </div>
+
+                {% if not is_past_due %}
+                    <input data-ng-disabled="answerForm.$invalid" type='button' class='ubcpi_submit' value="{{ 'Next Step' | translate }} &rarr;" name='ubcpi_next_step' data-ng-click="rc.clickSubmit($event);" aria-describedby="button-disabled-reason ubcpi-next-inline-hints" onclick=""/>
```

Comment:
This is the line with the "invalid filter: translate". This all shows to be new because I made a new templates directory and made a file